python/lsst/ip/diffim/getTemplate.py

```diff
@@ -126,46 +126,51 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         butlerQC.put(outputs, outputRefs)
 
     def getOverlappingExposures(self, inputs):
-        """Return lists of coadds and their corresponding dataIds that overlap
-        the detector.
+        """Return a data structure containing the coadds that overlap the
+        specified bbox projected onto the sky, and a corresponding data
+        structure of their dataIds.
+        These are the appropriate inputs to this task's `run` method.
 
         The spatial index in the registry has generous padding and often
         supplies patches near, but not directly overlapping the detector.
-        Filters inputs so that we don't have to read in all input coadds.
+        This filters the inputs so that we don't have to read in all
+        possibly-matching coadds.
 
         Parameters
         ----------
         inputs : `dict` of task Inputs, containing:
             - coaddExposures : `list` \
                               [`lsst.daf.butler.DeferredDatasetHandle` of \
                                `lsst.afw.image.Exposure`]
-                Data references to exposures that might overlap the detector.
+                Data references to exposures that might overlap the desired
+                region.
```

Comment:
I think I was sending it a dictionary containing a list of data references? I find info on the format needed to be useful but I defer to developers the correct way to document these things

Reply:
As noted in the docstring, this just takes a list of datarefs. See the type listing above this line.